operations/FAS_Loader/FAS_Loader_Fails.md

```diff
@@ -0,0 +1,67 @@
+## FAS LOADER FAILS
+
+**NOTE:** this procedure is only for csm-0.9.x releases.
+
+The FAS loader may fail due to issues with the `repomd.xml` file in Nexus.
+This will show up in the `fw-loader` job logs with the following message: `CRITICAL: Failed to get repomd.xml from repo`
+
+You may first notice the issue if firmware is not present in the FAS image list (`cray fas images list`) after running the FAS loader or if a firmware update reports `failed to find file, trying again soon`
+
+To view the FAS loader logs:

```

Comment:
This looks it should be steps with substeps. Change main step to "View the FAS loader logs."

It might also be important to include some more information on what to look for in the logs if that determines if the loader job needs to be run in the next step.

Reply:
Updated